### ISSUE TITLE
Norm and mean values...

### ISSUE THREAD

**Issue by xellDart** [opconty/keras-shufflenetV2#8]
Hi,thanks for your work, I have one question, for norm and mean values, is same as PyTorch? 

mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]

https://pytorch.org/hub/pytorch_vision_shufflenet_v2/

… or what values I need to use?

I follow this tutorial for this

https://www.kaggle.com/dcosmin/shufflenet-with-keras/comments

thanks
